pkg/assets/assets/monitoring/system-grafana-dashboard-1.json.tpl

```diff
@@ -69,21 +71,29 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sidekiq_job_runtime_seconds_count{namespace=\"$namespace\"}",
+          "expr": "sum(rate(sidekiq_jobs_success_total[1m]))",
```

Comment:
All expression should be "namespaced". Add `namespace` filter, should be something like this:
```
sum(rate(sidekiq_jobs_success_total{namespace='$namespace'}[1m]))
```